apps/condo/domains/billing/schema/BillingReceipt.js

```diff
@@ -90,6 +89,56 @@ const BillingReceipt = new GQLListSchema('BillingReceipt', {
             isRequired: true,
         },
 
+        formula: {
+            schemaDoc: 'Calculation formula. Example: balance + charge + recalculation + privilege + penalty',
+            type: Text,
+            // TODO(DOMA-6519): change read-only access to { create: true, read: true, update: true } after toPayDetails field removal
+            access: access.readOnlyAccess,
+        },
+
+        charge: {
+            ...MONEY_AMOUNT_FIELD,
+            schemaDoc: 'Amount of money charged by paid period. Example: "50.00", "-50.00"',
```

Comment:
You need to write about: what +50 is about return money, -50 related to paid money